composer.json

```diff
@@ -20,7 +20,7 @@
         "drupal/paragraphs": "^1.13",
         "drupal/tablefield": "^2.3",
         "drupal/viewsreference": "^2.0",
-        "localgovdrupal/localgov_core": "^2.0",
+        "localgovdrupal/localgov_core": "dev-feature/drupal-10 as 2.2.0",
```

Comment:
Doesn't want to go into the committed version. When localgov_core commit is merged ^2.0 will be good enough, dependency on D10 will make it have to install the latest version if you are installing D10